### DISCUSSION TITLE
Reset Password

### DISCUSSION THREAD

**Discussion by phrogg** [alienatedsec/yi-hack-v5#381] [Q&A] [answered]
Hey folks,

I cannot figure out my password. Is there a way to reset it via the microsd card data? I found the `passwd` file.

But I have no idea on how to go on.

Thanks in advance!

**Comment by alienatedsec** [comment]
Password to what? Web or console (e.g., SSH)?

**Reply by phrogg** [reply]
Ah sorry I thought they would be identical. To the web.

**Reply by alienatedsec** [reply, marked as answer]
Edit the`system.conf` file on the SD.

**Reply by phrogg** [reply]
Hey sorry, for coming back to you. I have the issue that my camera has some password set and It seems that I cannot reset it. I changed the passwords in the corresponding file `system.conf` but it does not accept it. Any other way or idea what I can do? Would be happy to donate if we get this working :)
